pyproject.toml

```diff
@@ -28,7 +28,7 @@ python = ">=3.8, <4.0"
 
 boto3 = "^1.24.11"
 botocore = "^1.27.11"
-pandas = "^1.2.0, <=1.5.1, !=1.5.0" # Exclusion per: https://github.com/aws/aws-sdk-pandas/issues/1678
+pandas = "^1.2.0, !=1.5.0" # Exclusion per: https://github.com/aws/aws-sdk-pandas/issues/1678
```

Comment:
We ve had problems with breaking changes in minor versions of pandas so we should at least pin it to `< 1.6` perhaps?